lib/kitchen/directions/bake_first_elements.rb

```diff
@@ -3,7 +3,13 @@
 module Kitchen
   module Directions
     module BakeFirstElements
-      def self.v1(within:, selectors:)
+      def self.v1(within:)
+        selectors = [
+          '.os-problem-container > .os-table',
```

Comment:
please add the tag names here as you can per https://openstax.slack.com/archives/C022NETP39N/p1621863752137000